lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -81,10 +77,9 @@ public Rotation2d fieldRelativeAngleToNote() {
      * @return A robot relative translational y value of an object in a camera in
      *         meters
      */
-    public Double getY() {
-        return m_cameraAngles.getY(m_notePosition24ArrayListener.getX().get(),
-                m_notePosition24ArrayListener.getY().get());
-    }
+    public double getY() {
```

Comment:
remove this.  callers can just get the Translation2d and pick out the x and y if they want to.